spec/models/firm_spec.rb

```diff
@@ -66,6 +66,37 @@
     end
   end
 
+  describe '#offices' do
+    let(:firm) { create(:firm) }
+    let!(:unsorted_offices) do
+      [
+        FactoryGirl.create(:office, firm: firm, address_line_one: 'd', created_at: Time.zone.now),
+        FactoryGirl.create(:office, firm: firm, address_line_one: 'b', created_at: 2.days.ago),
+        FactoryGirl.create(:office, firm: firm, address_line_one: 'a', created_at: 3.days.ago),
+        FactoryGirl.create(:office, firm: firm, address_line_one: 'c', created_at: 1.day.ago)
+      ]
+    end
+
+    describe 'default sort order' do
+      subject { firm.offices.map(&:address_line_one) }
+      it { is_expected.to eq(%w{a b c d}) }
```

Comment:
Just looking at the test it reads like the default sort order is based on `:address_line_one`

I'd suggest changing the values to `a => first, b => second, c=> third, d => fourth` as that would have the same meaning (as in this is the order offices should be returned in), however these values wouldn't be ordered alphabetically as ordering is by `:created_at`.